builder/googlecompute/artifact.go

```diff
@@ -142,6 +142,8 @@ func (a *Artifact) State(name string) interface{} {
 		return a.image.SizeGb
 	case "AccountFilePath":
 		return a.config.AccountFile
+	case "CredentialsFilePath":
+		return a.config.CredentialsFile
```

Comment:
I see that the `AccountFilePath` is kept in the artifact, but imho this may be problematic as this may expose credentials for other steps that come after it (typically post-processors over which we don't necessarily have control on), which could harvest them for nefarious purposes.